Internal/Debug App/Appetize Podfile/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '10.0'
+platform :ios, '13.0'
```

Comment:
The minimum should be 13.1 as stated in .podspec of the ThreeDS lib we are using.

Reply:
Updated, thanks!